version/version.go

```diff
@@ -4,4 +4,4 @@ const Maj = "0"
 const Min = "10"
 const Fix = "0"
 
-const Version = "0.10.0"
+const Version = Maj + "." + Min + "." + Fix
```

Comment:
Could you explain? 

Is the reason why we write it out as "0.10.0" that we grab the value from the dist.sh script?